youtube_dl/utils.py

```diff
@@ -2458,7 +2458,7 @@ def __init__(self, code=None, msg='Unknown error'):
 
         # Parsing code and msg
         if (self.code in (errno.ENOSPC, errno.EDQUOT)
-                or 'No space left' in self.msg or 'Disk quota excedded' in self.msg):
+                or 'No space left' in self.msg or 'Disk quota exceeded' in self.msg):
```

Comment:
This code has moved a number of times since its introduction, but it was introduced with this spelling, and I can find no support on the internet for looking for this spelling.

I suspect it was never noticed because the preceding line is doing the real work.

Reply:
This is error message for EDQUOT.
see: https://www-numi.fnal.gov/offline_software/srt_public_context/WebDocs/Errors/unix_system_errors.html

Reply:
That matches my correction: 
```
#define EDQUOT          122     /* Quota exceeded */
```
`Quota exceeded`, not the code as is. Thanks, that's good confirmation for this change